modules/gitops-argocd-cli-creating-an-application-in-default-mode.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+//
+// * argocd_applications/creating-an-application-using-gitops-argocd-cli.adoc
+// * declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+ifeval::["{context}" == "configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations"]
+:cluster:
+endif::[]
+ifeval::["{context}" == "creating-an-application-using-gitops-argocd-cli"]
+:app:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argocd-cli-creating-an-application-in-default-mode_{context}"]
+= Creating an application in the default mode by using the GitOps CLI
+
+You can create applications in the default mode by using the {gitops-shortname} `argocd` CLI.
+
+ifdef::cluster[]
+This sample workflow walks you through the process of configuring Argo CD to recursively sync the content of the `cluster` directory to the `cluster-configs` application. The directory defines the {OCP} cluster configurations and the `spring-petclinic` namespace on the cluster.
+endif::cluster[]
+
+.Prerequisites
+
+* You have installed the {gitops-title} Operator in your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the {gitops-title} `argocd` CLI.
+* You have logged into Argo CD instance.
+
+.Procedure
+
+. Get the admin account password for the Argo CD server:
```

Comment:
Should we use "administrator" rather than "admin"?

If it's a literal account name, I'd expect monospace formatting.